rocAL/rocAL/include/loaders/image/image_loader.h

```diff
@@ -83,6 +87,8 @@ class ImageLoader : public LoaderModule {
     size_t _image_counter = 0;//!< How many images have been loaded already
     size_t _remaining_image_count;//!< How many images are there yet to be loaded
     bool _decoder_keep_original = false;
+    bool _external_source_reader = false; //!< Set to true if external source reader
```

Comment:
We are setting this to false in the master graph also. Why are we setting it separately only for the image_loader?